src/modules/project/pages1/projectView/views/body/sections/header/components/LightningAddress.tsx

```diff
@@ -28,39 +27,39 @@ export const LightningAddress = ({ name, isGeyser, ...rest }: ILightningQR) => {
     setCopy(true)
     setTimeout(() => {
       setCopy(false)
-      onClose()
     }, 2000)
   }
 
   return (
-    <>
-      <Tooltip
-        label={copy ? t('Copied!') : t('Copy Lightning Address / Nostr identifier (NIP-05)')}
-        placement="top"
-        closeOnClick={false}
-        isOpen={isOpen}
+    <HStack w="full">
+      <Button
+        id="lightning-address"
+        size="md"
+        variant={copy ? 'solid' : 'soft'}
+        colorScheme="primary1"
+        leftIcon={<Icon as={PiLightning} fontSize={'18px'} />}
+        onClick={handleAddressCopy}
+        {...rest}
       >
-        <Button
-          id="lightning-address"
-          size="sm"
-          variant="soft"
-          colorScheme="primary1"
-          leftIcon={<PiLightning />}
-          onClick={handleAddressCopy}
-          onMouseEnter={onOpen}
-          onMouseLeave={onClose}
-          {...rest}
-        >
-          <Body size="sm" medium isTruncated flex={1}>
-            {name}
+        <Body size="md" medium isTruncated>
+          {name}
+        </Body>
+        {isGeyser && (
+          <Body size="md" medium>
+            {GEYSER_DOMAIN_POSTFIX}
           </Body>
-          {isGeyser && (
-            <Body size="sm" medium>
-              {GEYSER_DOMAIN_POSTFIX}
-            </Body>
-          )}
-        </Button>
-      </Tooltip>
-    </>
+        )}
+      </Button>
+      <Button
+        minWidth={24}
```

Comment:
Just by precaution here, did you mean `24` or `"24px"`

Reply:
it is 24.